packages/physics-physx/src/PhysXDynamicCollider.ts

```diff
@@ -79,13 +117,42 @@ export class PhysXDynamicCollider extends PhysXCollider implements IDynamicColli
     this._pxActor.setMassSpaceInertiaTensor(value);
   }
 
+  /**
+   * {@inheritDoc IDynamicCollider.getInertiaTensor }
+   */
+  getInertiaTensor(out: Vector3): Vector3 {
+    const inertia = this._pxActor.getMassSpaceInertiaTensor();
+    return out.set(inertia.x, inertia.y, inertia.z);
+  }
+
+  /**
+   * {@inheritDoc IDynamicCollider.setMassAndUpdateInertia }
+   */
+  setMassAndUpdateInertia(mass: number): void {
+    this._pxActor.setMassAndUpdateInertia(mass);
+  }
+
+  /**
+   * {@inheritDoc IDynamicCollider.getMaxAngularVelocity }
+   */
+  getMaxAngularVelocity(): number {
+    return this._pxActor.getMaxAngularVelocity();
+  }
```

Comment:
If never used, can delete to reduce wasm size

Reply:
done